cppwinrt/cmd_reader.h

```diff
@@ -474,7 +474,11 @@ namespace cppwinrt
                         xml_path = item.path() / sdk_version;
                         xml_path /= L"SDKManifest.xml";
 
-                        add_files_from_xml(files, sdk_version, xml_path, sdk_path);
+                        // Not all Extension SDKs include an SDKManifest.xml file; ignore those which do not (e.g. WindowsIoT).
+                        if (std::filesystem::is_regular_file(xml_path))
+                        {
+                            add_files_from_xml(files, sdk_version, xml_path, sdk_path);
```

Comment:
I assume an exception happens when this is missing. perhaps handling that is better.

Reply:
There are only a handful of `catch` statements in this codebase, and all of them either re-throw, or print an error message before terminating, so I'm hesitant to introduce control flow logic based on exception handling here.

I did consider testing the return value of `SHCreateStreamOnFileW` against `HRESULT_FROM_WIN32(ERROR_PATH_NOT_FOUND)` and returning early from `add_files_from_xml()` rather than throwing, but I felt it was clearer to have this logic at the call site where the paths are built.

Reply:
Checking first seems like a pretty obvious TOCTOU bug

Reply:
Fair point in general, and I'm happy to switch to reacting to failure instead if that's preferred; however in this case such a bug would require someone to be uninstalling the Windows SDK *while* running cppwinrt.exe, which seems quite impractical/unlikely.

Reply:
I like that idea (and the `ignore_if_missing` parameter is important since this function is also used for `Platform.xml` which *should* always be present). Update incoming...